diff --git a/node_modules/jsonschema/lib/validator.js b/node_modules/jsonschema/lib/validator.js
index cca5e9c..8d39381 100644
--- a/node_modules/jsonschema/lib/validator.js
+++ b/node_modules/jsonschema/lib/validator.js
@@ -260,7 +260,7 @@ Validator.prototype.resolve = function resolve (schema, switchSchema, ctx) {
     return {subschema: ctx.schemas[switchSchema], switchSchema: switchSchema};
   }
   // Else try walking the property pointer
-  let parsed = new URL(switchSchema,'thismessage::/');
+  let parsed = new URL(switchSchema,'thismessage:///');
   let fragment = parsed.hash;
   var document = fragment && fragment.length && switchSchema.substr(0, switchSchema.length - fragment.length);
   if (!document || !ctx.schemas[document]) {
